.ci/updatecli/updatecli-bump-stack-version.yml

```diff
@@ -40,7 +40,7 @@ actions:
     scmid: githubConfig
     sourceid: lastStableStackSnapshot
     spec:
-      automerge: true
+      automerge: false
```

Comment:
We cannot use `automerge` here, the branch protections have a higher precedence and the `GitHub Actions [bot]` does not support it yet, see https://github.com/orgs/community/discussions/13836.


```
Pull Request available at:

	https://github.com/elastic/elastic-agent/pull/8576

ERROR: action stage:	"Merge method merge commits are not allowed on this repository and Pull request User is not authorized for this protected branch"
Existing GitHub pull request found: https://github.com/elastic/elastic-agent/pull/8576
ERROR: running actions:
errors occurred while running actions:
	* Merge method merge commits are not allowed on this repository and Pull request User is not authorized for this protected branch

=============================
```